mysite/settings.py

```diff
@@ -0,0 +1,119 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 2.0.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 's_#$5r*&&pn-p+2@ofesbsgf3pr4ftfy7p#8%c2y=-du@u$6+z'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['127.0.0.1', 'serhii73.pythonanywhere.com']
```

Comment:
Why hardcode things? Why do smth deployment-related in initial PR?

Reply:
Sorry. Just because it was written in the tutorial.
I understand that this is not correct.
I will fix it.